handlers/handlers.go

```diff
@@ -25,29 +39,157 @@ func setStatusCode(req *http.Request, w http.ResponseWriter, err error) {
 	w.WriteHeader(status)
 }
 
-// HelloWorld Handler
-func HelloWorld(cfg config.Config) http.HandlerFunc {
+// getCookiePreferencePage talks to the renderer to get the cookie preference page
+func getCookiePreferencePage(ctx context.Context, w http.ResponseWriter, req *http.Request, rendC RenderClient, cp cookies.Policy) error {
+	var err error
+	m := mapper.CreateCookieSettingPage(cp)
+	b, err := json.Marshal(m)
+	if err != nil {
+		log.Event(ctx, "unable to marshal cookie preferences", log.Error(err))
+		setStatusCode(req, w, err)
+		return err
+	}
+
+	templateHTML, err := rendC.Do("cookies-preferences", b)
+	if err != nil {
+		log.Event(ctx, "getting template from renderer cookies-preferences failed", log.Error(err))
+		setStatusCode(req, w, err)
+		return err
+	}
+	if _, err := w.Write(templateHTML); err != nil {
+		log.Event(ctx, "error on write of cookie template", log.Error(err))
+		setStatusCode(req, w, err)
+	}
+	return err
+}
+
+// isProtectedCookie is a helper function that checks if a cookie is protected or not
+func isProtectedCookie(stringToFind string, slice [5]string) bool {
```

Comment:
Would it be better to use the name `cookies` instead of `slice`?

Reply:
Do you need to pass the slice as an arg? If this will always be the `protectedCookies` just use the package var instead.